tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/package.scala

```diff
@@ -13,4 +13,8 @@ package object tensorflow {
   implicit def nd2Tensor(nd: NDArray[Float]): Tensor = {
     nd.as[Tensor]
   }
+
+  implicit def seq2Tensor(s: Seq[Int]): Tensor = {
```

Comment:
Hmm what's the use case for this one as opposed to `Tensor#seq2Tensor`?

Reply:
look at the package this is how it was done in old api - scala 2. 
I just added one more converter. 

Why I added this? 
Because we need conversion Seq[Int] -> Old Tensor (and this is the place we have those)
In Tensor object in new api we have conversion Seq[Int] -> New Tensor (just in case)

Reply:
Oh ok, I see now